README.md

```diff
@@ -126,9 +128,6 @@ public sealed class MainActivity : Microsoft.UI.Xaml.ApplicationActivity
 Note that when you run your app in debug from Visual Studio (or other IDEs), the new SplashScreen icon doesn't show.
 It shows when you run the app from the launcher (even debug builds).
 
```

Comment:
Wondering if there are any additional features we want to detail here that not covered in the Getting Started section?

Reply:
To be honest I would remake it like this one https://github.com/nventive/ReviewService

Reply:
@jeanplevesque

Reply:
It's fine like this. :)

Also, I don't think there are other features to document.